src/watch.rs

```diff
@@ -11,11 +11,14 @@ use std::{
 #[non_exhaustive]
 #[derive(Debug, Parser)]
 pub struct Watch {
+    /// Watch specific file(s) or folder(s). Default to workspace root
     #[clap(long = "watch", short = 'w')]
     pub watch_paths: Vec<PathBuf>,
+    /// Paths that will not be monitored for changes. Note that the target directory or hidden
```

Comment:
> Note that the target directory or hidden

This explanation does not belong here. It should be placed on the struct's documentation or the run function. In this case the struct's documentation is used by clap so it's best to move it to the run function instead.

Why it doesn't belong here? Because it describes a behavior of the object (`Watch`) itself. It's not a detail about the field itself.

```suggestion
    /// Paths that will excluded.
```